packages/worker/src/interceptors.ts

```diff
@@ -44,6 +66,10 @@ export interface WorkerInterceptors {
    * List of factory functions returning {@link ActivityInboundCallsInterceptor}s
    */
   activityInbound?: ActivityInboundCallsInterceptorFactory[];
+  /**
+   * List of factory functions returning {@link ActivityOutboundCallsInterceptor}s
+   */
+  activityOutbound?: ActivityOutboundCallsInterceptorFactory[];
```

Comment:
Let's change this to:

```
activityInbound?: ActivityInboundCallsInterceptorFactory[]; // <-- deprecate this
activity?: ActivityInterceptorsFactory[]; <-- same as the workflow counterpart
```

Reply:
It'll help you reference the outbound from the inbound in case they need to interact (and reuse the same implementation for both).

Reply:
Done